lib/ex_unit/test/ex_unit_test.exs

```diff
@@ -720,4 +799,9 @@ defmodule ExUnitTest do
   defp max_failures_reached_msg() do
     "--max-failures reached, aborting test suite"
   end
+
+  # Strips escape-sequence colors from IO.ANSI strings
+  defp strip(string) do
+    String.replace(string, ~r/\e\[(\d+;?)+m/, "")
+  end
```

Comment:
We don't need this strip function. :)

Reply:
the invalid counter shows these codes, how to do check for that?

Reply:
Are you suggesting doing something like this?
https://github.com/elixir-lang/elixir/blob/b611983d57258a2cff82de48216e9f66bd566268/lib/ex_unit/test/ex_unit_test.exs#L356-L358

the problem that approach is that we are not seeing the full line in the test.
Actually, I am thinking about submitting a new PR after this that checks for the output message until the end of the line (including `\n`). Have we had theses checks, they bug that missed the skipped messages wouldn't have been introduced.

Reply:
Can't we disable ansi codes when running the test suite?

Reply:
Didn't think of it.
fixed!